mne/label.py

```diff
@@ -747,6 +747,9 @@ def center_of_mass(self, subject=None, restrict_vertices=False,
         subject = _check_subject(self.subject, subject)
         if np.any(self.values < 0):
             raise ValueError('Cannot compute COM with negative values')
+        if np.all(self.values == 0):
+            raise RuntimeWarning('Label values are all 0, treating as a structural label\n')
+            self.values = np.ones(self.values.shape)
```

Comment:
changing the objects attributes here is dangerous.

besides ok for the change but you need to update/clarify the docstring and document the change in what's new.